src/asmCrashReport.cpp

```diff
@@ -237,7 +245,12 @@ namespace asmCrashReport
 
       if ( inExceptionInfo->ExceptionRecord->ExceptionCode == EXCEPTION_STACK_OVERFLOW )
       {
+          // https://stackoverflow.com/a/38019482
+#ifdef _M_IX86
          frameInfoList += _addr2line( sProgramName, reinterpret_cast<void*>(inExceptionInfo->ContextRecord->Eip) );
+#else

```

Comment:
Could you please use `#elif _M_X64` and make the `#else` produce an `#error` saying this needs to be implemented (as above).

Reply:
Ok